neuro_extras/main.py

```diff
@@ -255,11 +268,8 @@ async def _run_copy_container(src_cluster: str, src_path: str, dst_path: str) ->
         f'"cp --progress -r -u -T storage:{src_path} /storage/{dst_path}"',
     ]
     cmd = " ".join(args)
-    print(f"Executing '{cmd}'")
-    subprocess = await asyncio.create_subprocess_shell(cmd)
```

Comment:
please note there was `create_subprocess_shell` and now it's `create_subprocess_exec`